themes/ddbasic/sass/p2/p2-rating.scss

```diff
@@ -92,6 +92,17 @@
           color: $color-secondary;
         }
       }
+
+      .material-item.odd &,
+      .ting-object.view-mode-teaser &,
+      .ting-object.view-mode-search-result & {
+        &.active,
+        &.submitted {
+          &::after {
+            color: $color-secondary;
+          }
+        }
+      }
```

Comment:
This really needs some refactoring but I guess we will have to let it pass for now.